storage/shard.go

```diff
@@ -720,6 +751,20 @@ func (s *dbShard) Write(
 	value float64,
 	unit xtime.Unit,
 	annotation []byte,
+) error {
+	return s.writeAndIndex(ctx, id, ident.EmptyTagIterator, timestamp,
+		value, unit, annotation, nil)
+}
+
+func (s *dbShard) writeAndIndex(
```

Comment:
This function is pretty similar to the write() function, but I assume its hard to DRY it up? Maybe just leave a comment on them that if you make a change to one to look at the other

Reply:
I don't follow. `Write()` and `WriteTagged` both call `writeAndIndex()` internally for re-use.

Which write() are you talking about?

Reply:
Looking back at the code, I have no idea what I was talking about. Maybe I was looking at an old version or something

Reply:
Lol fair enough